### PR TITLE
feat: expose inflight singleton as a Debouncer class prop, test debounce method

### PR DESCRIPTION
**Pull request by patrickocoffeyo**

### CODE REVIEW

**Comment by patrickocoffeyo** [comment]
@e0ipso any thoughts on this?

File: src/Debouncer.js [hunk 1]
```diff
@@ -10,6 +10,14 @@ import type { DebouncerInterface, InflightStore } from '../types/common';
 const inflight: InflightStore = new Map();
```
Comment (e0ipso) [inline src/Debouncer.js]: Do we want to remove this then?

**Comment by e0ipso** [comment]
@patrickocoffeyo can you explain the use case for exposing this?

In any case please feel free to self merge (or ask Elliott to do so), since this was approved by @elliotttf.

**Comment by patrickocoffeyo** [comment]
@e0ipso I've been doing some investigative work around a memory leak. I don't _think_ it involves this variable, but I thought it may make sense to expose `inflight` so that it could be accessed from instances of `Debouncer`. This also enabled me to add another test and make some new assertions around how `inflight` is managed, which I thought might be useful.

Neither I nor Elliott have committer access to this repository, but no worries, if you don't see the use case I'm happy for this to be closed! :)

**Pull request merged**